### ISSUE TITLE
Thousand Land only rendering black screen

### ISSUE THREAD

**Issue by Kravataf** [Cxbx-Reloaded/Cxbx-Reloaded#2459]
### System Configuration:

* OS - windows 11
* CPU - ADM ryzen 7 5800H with radeon graphics
* GPU0 - NVIDIA GEFORCE RTX 3060 laptop GPU
* GPU1 - ADM Radeon(TM) graphics

**Comment by Catcher40** [comment]
I don't think it's just Thousand Land, ever since I did the update to the version dated 7th June every game I have tried just renders a black screen. It doesn't matter what options I change I always get the same black screen.

I'm using Windows 11 x64 Pro
Asus x570 Plus MB with a Ryzen 7 5800x CPU oc@4.6
RTX 4700 TI GPU, 4 x 8 gig Corsair dominator Memory @3600.

**Comment by Inno2d** [comment]
Yup, same here. Updated to latest build and black screen on every game.

On windows 10 pro 22H2 with Ryzen 5600x@4.6, RTX 3050

**Comment by xMoJox12** [comment]
SAME. Just stick with the previous version I guess lol

**Comment by Triticum0** [comment]
This game uses NaN handling for rendering so wouldn't expect it to render much. same engine as Otogi

**Comment by Kravataf** [comment]
> This game uses NaN handling for rendering so wouldn't expect it to render much. same engine as Otogi

well idk much about emulating but it was made by the same studio + around the same time.. idk though the xemu issue may be completely unrelated

**Comment by Catcher40** [comment]
I checked and I don't have any hacks enabled not only that but games that I knew worked like Buffy the Vampire Slayer 1 or Genma Onimusha just give me the black screen with the audio playing, I've noticed that you can still use the controller to select various main menu options as it plays the sound when you move around or press button A it just seems to refuse to show any graphics.
